tools/csv-merger/csv-merger.go

```diff
@@ -456,7 +456,9 @@ func generateUnifiedCSV() *csvv1.ClusterServiceVersion {
 			ocsCSV.Spec.CustomResourceDefinitions.Owned[i].Description = internalCRDDescription + "StorageCluster Initialization represents a set of tasks the OCS operator wants to implement for every StorageCluster it encounters."
 		// backingstore and bucketclass can be used by the admin, so avoid adding internal prefix to these resources
 		case "backingstores.noobaa.io":
+			ocsCSV.Spec.CustomResourceDefinitions.Owned[i].Description = "Backing Stores are storage targets that are used to store chunks of data on MCG buckets."
 		case "bucketclasses.noobaa.io":
+			ocsCSV.Spec.CustomResourceDefinitions.Owned[i].Description = "BucketClass is a CRD representing a class for buckets that defines tiering policies and data placements for an OBC on MCG buckets."
```

Comment:
Got an input from PM/Marketing, we should not use acronyms

```suggestion
			ocsCSV.Spec.CustomResourceDefinitions.Owned[i].Description = "BucketClass is a CRD representing a class for buckets that defines tiering policies and data placements for object buckets."
```

Reply:
Changed the description according to the above comment...